rpc/handler.go

```diff
@@ -334,8 +334,19 @@ func (h *handler) handleCall(cp *callProc, msg *jsonrpcMessage) *jsonrpcMessage
 		return msg.errorResponse(&invalidParamsError{err.Error()})
 	}
 	start := time.Now()
-	answer := h.runMethod(cp.ctx, msg, callb, args)
+	answerCh := make(chan *jsonrpcMessage)
```

Comment:
This channel needs to be buffered, otherwise the handler goroutine will leak when there is a timeout.